fastlane/Fastfile

```diff
@@ -129,12 +129,32 @@ platform :ios do
   ENV['FASTLANE_XCODEBUILD_SETTINGS_TIMEOUT'] = '120'
 
   desc 'Run the unit tests'
-  lane :test do
+  lane :test do |options|
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Automattic/pocket-casts-ios/blob/0d9adb5eba61feebfcfef9fdfef2a2ae16d90e3d/fastlane/Fastfile#L132"></span>Lint/UnusedBlockArgument: Unused block argument - <code>options</code>. You can omit the argument if you don't care about it.</td>
    </tr>
  </tbody>
</table>